pkg/install/0-installstorage.go

```diff
@@ -169,7 +169,7 @@ func (i *Installer) installStorage(ctx context.Context, installConfig *installco
 		}
 
 		i.log.Print("deploying storage template")
-		err = i.deployments.CreateOrUpdateAndWait(ctx, i.doc.OpenShiftCluster.Properties.ResourceGroup, "azuredeploy", mgmtresources.Deployment{
+		err = i.deployments.CreateOrUpdateAndWait(ctx, i.doc.OpenShiftCluster.Properties.ResourceGroup, "installstorage", mgmtresources.Deployment{
```

Comment:
can revert this, I don't think it's needed.

Reply:
This makes debugging little big cleaner, dont you think so?